www/js/lib/zimfile.js

```diff
@@ -21,6 +21,32 @@
  */
 'use strict';
 
+/**
+ * Add Polyfill currently required by IE11 to run zstddec-asm and xzdec-asm
+ * See https://github.com/emscripten-core/emscripten/issues/14700
+ * If this is resolved upstream, remove this polyfill
+ * Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith
+ */
+    if (!String.prototype.startsWith) {
+    Object.defineProperty(String.prototype, 'startsWith', {
+        value: function(search, rawPos) {
+            var pos = rawPos > 0 ? rawPos|0 : 0;
+            return this.substring(pos, pos + search.length) === search;
+        }
+    });
+}
+
+/**
+ * A global variable to track the assembler machine type and the last used decompressor (for reporting to the API panel)
+ * This is populated in the Emscripten wrappers
+ * @type {Object}
```

Comment:
Add object properties to type definition?

Reply:
```
     * @property {String} assemblerMachineType The assembler machine type supported and/or loaded by this app: 'ASM' or 'WASM'
     * @property {String} decompressorLastUsed The decompressor that was last used to decode a compressed cluster (currently 'XZ' or 'ZSTD')
     * @property {String} errorStatus A description of any detected error

```

Reply:
Good idea